src/azul/queues.py

```diff
@@ -178,7 +178,10 @@ def wait_for_queue_level(cls, queue_names, empty: bool = True, timeout: int = 60
         wait_start_time = time.time()
         queue_size_history = deque(maxlen=10)
 
-        logger.info('Waiting up to %s seconds for %s queues to empty ...', timeout, len(queues))
+        logger.info('Waiting up to %s seconds for %s queues to %s ...',
+                    timeout,
```

Comment:
```suggestion
                    timeout, len(queues), 'empty' if empty else 'not be empty')
```

For log statements, I would actually prefer to exclude the first argument from the wrap all or none rule:

https://github.com/DataBiosphere/azul/pull/1479/files